scheduler/cups-driverd.cxx

```diff
@@ -2914,3 +2927,74 @@ regex_string(const char *s)		/* I - String to compare */
 
   return (NULL);
 }
+
+void *
+cupsFileGetsCall(void *input)
+{
+  retval = cupsFileGets(((struct args*)input)->fp,((struct args*)input)->buf,((struct args*)input)->buflen);
+  return NULL;
+}
+
+/*
+  * Alternate to cupsFileGets() 
+  */
```

Comment:
@pranjanpr would you mind expanding the comment? Please look into other comments for CUPS functions.
Indentation doesn't match, would you mind looking into it?